app/models/product.rb

```diff
@@ -23,7 +22,10 @@ class Product < ActiveRecord::Base
   default_scope { includes(:general_product) }
 
   # transient properties for conveniently adding sibling (product) units
-  attr_accessor :sibling_id, :sibling_name, :sibling_short_description, :sibling_long_description, :sibling_organic, :sibling_unit_id, :sibling_unit_description, :sibling_product_code, :sibling_unit_quantity, :skip_validation, :consignment_market
+  attr_accessor :sibling_id, :sibling_name, :sibling_short_description,
+    :sibling_long_description, :sibling_organic, :sibling_unit_id,
+    :sibling_unit_description, :sibling_product_code, :sibling_unit_quantity,
+    :skip_validation, :consignment_market
```

Comment:
@thermistor seems like this sibling junk is just to enable addition of a unit from the product view yes? Super funky. This whole general_product <-> product thing is pretty messed up as implemented but we can leave it alone and  fight it another day.